src/commands/sns_summary.rs

```diff
@@ -0,0 +1,31 @@
+use crate::commands::sns_canisters_summary::SnsCanistersSummaryOps;
+use crate::commands::{nervous_system_parameters, sns_canisters_summary};
+use crate::lib::signing::IngressWithRequestId;
+use crate::lib::AnyhowResult;
+use crate::SnsCanisterIds;
+use clap::Parser;
+
+#[derive(Parser, Debug)]
```

Comment:
Same here: I didn't see a debug use of this command anywhere. Is this here for a special reason?